src/pybind/mgr/dashboard/frontend/src/app/ceph/rgw/rgw-user-form/rgw-user-form.component.html

```diff
@@ -80,8 +80,7 @@
                formControlName="display_name"
                name="display_name"
                id="display_name"
-               [invalid]="userForm.controls.display_name.invalid && (userForm.controls.display_name.dirty)"
-               [readonly]="editing"/>
+               [invalid]="userForm.controls.display_name.invalid && (userForm.controls.display_name.dirty)"/>
```

Comment:
we can use showError() here
```suggestion
               [invalid]="userForm.showError('display_name', formDir)"/>
```